scripts/create-releases.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Can you run this through shellcheck? I think we're going to be linting files that way soon.